crates/topos-sequencer-subnet-client/src/lib.rs

```diff
@@ -295,10 +283,11 @@ impl SubnetClient {
         cert: &Certificate,
         cert_position: u64,
     ) -> Result<Option<TransactionReceipt>, Error> {
+        // Tx Root from the certificate is currently not used by the smart contract
```

Comment:
True that it is not being used in the smart contracts, but we still need to pass it as it might be used later on to verify the zk proofs.